content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Jaeger
+description: How to configure the proxies to send tracing requests to Jaeger.
+weight: 10
+keywords: [telemetry,tracing,jaeger]
+---
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+    Either use the `istio-demo.yaml` or `istio-demo-auth.yaml` template, which includes tracing support, or
+    use the Helm chart with tracing enabled by setting the `--set tracing.enabled=true` option.
+
+* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+## Accessing the dashboard
+
+Setup access to the tracing dashboard by using port-forwarding:
```

Comment:
Please use numbered steps in procedures. Numbered lists allow readers to follow the instructions much easier and to keep their place on the process. Avoid gerunds, the time space associated with that conjugation in English does not exist in other languages making the content a lot harder to grasp for non English native speakers. Use simple present instead. This comment applies to all prepositional phrases following the construct: by ... verb-ing
```suggestion
1. To set up access to the tracing dashboard, use port forwarding:
```